dependencies.props

```diff
@@ -47,17 +50,18 @@
     
     <XmlUpdateStep Include="CoreFx">
       <Path>$(MSBuildThisFileFullPath)</Path>
-      <ElementName>PlatformPackageVersion</ElementName>
-      <PackageId>Microsoft.NETCore.Platforms</PackageId>
+      <ElementName>CoreFxExpectedPrerelease</ElementName>
```

Comment:
Sounds good. I haven't worked with this stuff too often so I'm not up-to-date with the best practices. I'll do what you're suggesting.